core/src/services/fs/backend.rs

```diff
@@ -292,12 +292,12 @@ impl FsBackend {
 
 #[async_trait]
 impl Accessor for FsBackend {
-    type Reader = oio::into_reader::FdReader<Compat<tokio::fs::File>>;
+    type Reader = oio::into_reader::FdReader<Compat<fs::File>>;
```

Comment:
I perfer to use `tokio::fs::File` here. We can remove the `use tokio::fs` instead.